src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,212 @@
+package storage;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
```

Comment:
Is it better to use "import java.io.*" since there are more than three import under "java.io"?

Reply:
I checked the Java coding standard and they state that "Imported classes should always be listed explicitly" and I cannot find the statement about using * when there is more than three import. Is it stated at any other place which I might have missed out?

Reply:
Sorry, I think I have remembered it wrongly :(. Thanks!